pkg/deploy/api/types.go

```diff
@@ -118,6 +123,14 @@ type ExecNewPodHook struct {
 	Volumes []string
 }
 
+// TagImageHook is a request to tag the image in a particular container onto an ImageStreamTag.
+type TagImageHook struct {
+	// ContainerName is the name of a container in the deployment config whose image value will be used as the source of the tag
+	ContainerName string
```

Comment:
Is defaultable, don't see it being necessary to default it.  I'd rather
people be explicit.  However, I will validate it for people

On Thu, Feb 11, 2016 at 8:24 AM, David Eads <notifications@github.com>
wrote:

> In pkg/deploy/api/types.go
> <https://github.com/openshift/origin/pull/6925#discussion_r52599998>:
>
> > @@ -118,6 +123,14 @@ type ExecNewPodHook struct {
> >  	Volumes []string
> >  }
> >
> > +// TagImageHook is a request to tag the image in a particular container onto an ImageStreamTag.
> > +type TagImageHook struct {
> > +	// ContainerName is the name of a container in the deployment config whose image value will be used as the source of the tag
> > +	ContainerName string
>
> This is defaultable when we only have one container. I don't see that
> happening and it should be mentioned in the godoc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6925/files#r52599998>.
>